.github/workflows/functional_all_db.yml

```diff
@@ -21,6 +21,14 @@ env:
 jobs:
   test_functional_mvc:
     name: Functional Tests - MVC
+    # These conditions are needed because the workflow triggers on individual PR comments too, see
```

Comment:
```suggestion
    # These conditions are needed because the workflow triggers individual PR comments too, see
```

Reply:
It doesn't trigger comments ("trigger comments" ~= "creates comments") but is triggered by comments being added, hence [the GitHub terminology "on"](https://github.com/OrchardCMS/OrchardCore/blob/92550c2518ce22814d91c66359ceb41c2db7429e/.github/workflows/functional_all_db.yml#L3).